### PR TITLE
Update cookie expiration time every day

### PR DESCRIPTION
**Pull request by jridgewell**
Similar to how we extend the expiration time of our LocalStorage CIDs, we now
extend the expiration time of the Cookie CIDs.

I did not implement a check to prevent multiple updates per day, since cookie
setting is inherently faster than localstorage.

Closes https://github.com/ampproject/amphtml/issues/2272.

### CODE REVIEW

File: src/service/cid-impl.js [hunk 1]
```diff
@@ -267,20 +288,16 @@ function store(win, cidString) {
  * @return {!BaseCidInfoDef|undefined}
  */
 function read(win) {
-  let val;
   try {
-    val = win.localStorage.getItem('amp-cid');
+    const data = win.localStorage.getItem('amp-cid');
+    const item = JSON.parse(data);
```
Comment (cramforce) [inline src/service/cid-impl.js]: why this change?

If this is not JSON we'd want to know in error reporting.

Comment (jridgewell) [inline src/service/cid-impl.js]: Ah, I can revert this then.

Comment (jridgewell) [inline src/service/cid-impl.js]: Fixed.

**Comment by cramforce** [comment]
Did you manually test this?

**Comment by jridgewell** [comment]
Yup, working as intended.

**Comment by cramforce** [comment]
LGTM
